packages/appsync-modelgen-plugin/src/visitors/appsync-json-metadata-visitor.ts

```diff
@@ -189,11 +192,17 @@ export class AppSyncJSONVisitor<
     }));
   }
   private generateModelMetadata(model: CodeGenModel): JSONSchemaModel {
+    const pluralAttributes = this.config.improvePluralization ?
+      {
+        listPluralName: plurality(model.name, this.config.improvePluralization),
+        syncPluralName: this.pluralizeModelName(model)
```

Comment:
should json syncPluralName be different than java visitor? 
https://github.com/aws-amplify/amplify-codegen/pull/255/files#diff-8fdb1b5aed85eca745295520073edb2af47d27fdf8cff3173487f147b11e5fdaR761

Reply:
good catch I think the java visitor needs fixed